patches/net/minecraft/world/level/block/BubbleColumnBlock.java.patch

```diff
@@ -0,0 +1,24 @@
+--- a/net/minecraft/world/level/block/BubbleColumnBlock.java
++++ b/net/minecraft/world/level/block/BubbleColumnBlock.java
+@@ -121,10 +_,10 @@
+     private static BlockState getColumnState(BlockState p_152718_) {
+         if (p_152718_.is(Blocks.BUBBLE_COLUMN)) {
+             return p_152718_;
+-        } else if (p_152718_.is(Blocks.SOUL_SAND)) {
++        } else if (p_152718_.sustainBubbleColumn() == net.neoforged.neoforge.common.enums.BubbleColumnDirection.UPWARD) { // Neo: PR #931
```

Comment:
I would prefer a local variable for the direction instead of retrieving it twice.

Reply:
Was trying to keep patch smaller but one extra line shouldn't be too bad.